iothub/service/src/Common/Security/ISharedAccessSignatureCredential.cs

```diff
@@ -9,7 +9,7 @@ namespace Microsoft.Azure.Devices.Common.Security
     /// <summary>
     /// Credential interface used for authentication and authorization.
     /// </summary>
-    public interface ISharedAccessSignatureCredential
+    internal interface ISharedAccessSignatureCredential
```

Comment:
again, this interface is not referenced anywhere in our public API, so I made it internal